recipe/meta.yaml

```diff
@@ -52,13 +57,17 @@ outputs:
         - google.api_core.retry
         - google.api_core.timeout
       commands:
-        - python -m pip check
+        - pip check
 
   - name: {{ name }}-grpc
     build:
-      noarch: generic
+      # noarch: generic
+      skip: true  # [s390x]
     requirements:
+      host:
+        - python
```

Comment:
Adding `python` as a build requirement to the metapackages is necessary to ensure that the metapackages are built using the same build matrix as the main.